### ISSUE TITLE
New Release?!?

### ISSUE THREAD

**Issue by jedie** [khertan/ForRunners#57]
I have seen some new commits. But there is no new release here and not via F-Droid.

I'd like to see a new version available ;)

**Comment by jedie** [comment]
ping

**Comment by jedie** [comment]
ping

**Comment by jedie** [comment]
ping

**Comment by wkyle** [comment]
Agreed. The version on the repo is much more polished. The older version on FDroid does a disservice to the current (excellent) state of the app. Time for an updated release.

**Comment by brvier** [comment]
Hi,

The version on the repository is far more advanced, but for a reason I didn’t understand it crash at startup on two of my Android device ... Sony z1c and a Asus Zen Phone 3 while it s works on my Galaxy A3.

If some of you can confirm working one z1c or Zen Phone 3 ... I ll push it

Regards 

Benoît HERVIER - http://rvier.fr

**Comment by jedie** [comment]
Where can i download a "pre-release" apk for testing?

IMHO github can host the apk's, too.

**Comment by jedie** [comment]
ping:

> Where can i download a "pre-release" apk for testing?

**Comment by brvier** [comment]
1.2.1 Beta 1 Release, please make a **backup before**.

https://github.com/khertan/ForRunners/releases/tag/1.2.1beta1

**Comment by jedie** [comment]
I used 1.2.1beta1 and it works fine on my HTC 10 With LineageOS 14.1-20180331 (Android 7.1.2)

The startup seems to be lower than the previous release and i miss the GPX import function :(

**Comment by brvier** [comment]
The gpx import is still here, it just automatically discover the file type (gpx, forrunners session, or other formart)

**Comment by brvier** [comment]
1.2.1beta seems to crash on Xiaomi Device #71
